python/doc/examples/data_analysis/quick_start_point_and_sample.ipynb

```diff
@@ -0,0 +1,992 @@
+      ]
+     },
+     "execution_count": 13,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "type ( column )"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We see that:\n",
+    "\n",
+    "* the `row` is a `Point`,\n",
+    "* the `column` is a `Sample`.\n",
+    "\n",
+    "This is consistent with a fact that, in a dimension $D$ `Sample`, a row is a $D$-dimensional `Point`."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The following statement extracts several columns (with indices 0 and 2) and creates a new `Sample`. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<TABLE><TR><TD></TD><TH>v0</TH><TH>v1</TH></TR>\n",
+       "<TR><TD>0</TD><TD>0.0</TD><TD>0.0</TD></TR>\n",
+       "<TR><TD>1</TD><TD>0.0</TD><TD>0.0</TD></TR>\n",
+       "<TR><TD>2</TD><TD>0.0</TD><TD>0.0</TD></TR>\n",
+       "<TR><TD>3</TD><TD>0.0</TD><TD>32.0</TD></TR>\n",
+       "<TR><TD>4</TD><TD>0.0</TD><TD>0.0</TD></TR>\n",
+       "</TABLE>"
+      ],
+      "text/plain": [
+       "class=Sample name=Unnamed implementation=class=SampleImplementation name=Unnamed size=5 dimension=2 data=[[0,0],[0,0],[0,0],[0,32],[0,0]]"
+      ]
+     },
+     "execution_count": 14,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "data.getMarginal ([0 , 2])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Create a `Point` or a `Sample` from a Python list\n",
+    "\n",
+    "The following statement creates a `Point` from a Python list."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<p>[2,3]</p>"
+      ],
+      "text/plain": [
+       "class=Point name=Unnamed dimension=2 values=[2,3]"
+      ]
+     },
+     "execution_count": 15,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "p1 = ot.Point ([2 , 3])\n",
+    "p1"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<p>[0,1]</p>"
+      ],
+      "text/plain": [
+       "class=Point name=Unnamed dimension=2 values=[0,1]"
+      ]
+     },
+     "execution_count": 16,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "p2 = ot.Point(range(2))\n",
+    "p2"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The first useful *Pythonism* that we will review is the *list comprehension*. This allows to create a list from a `for` loop. This kind of statements is often used in the the examples, so that they can be as short as possible.\n",
```

Comment:
```suggestion
    "The first useful *Pythonism* that we will review is the *list comprehension*. This makes it possible to create a list from a `for` loop. This kind of statements is often used in the the examples, so that they can be as short as possible.\n",
```

Reply:
> 

The construction ["this allows to"](https://english.stackexchange.com/questions/85069/is-the-construction-it-allows-to-proper-english/103971) is not used by native speakers.

Reply:
> 

The construction ["this allows to"](https://english.stackexchange.com/questions/85069/is-the-construction-it-allows-to-proper-english/103971) is not used by native speakers.

Reply:
Ok, I did'n t know that. Replaced by the simpler "This creates..."